jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -652,6 +652,8 @@ private InterpreterResult executeSql(String propertyKey, String sql,
       for (int i = 0; i < sqlArray.size(); i++) {
         String sqlToExecute = sqlArray.get(i);
         statement = connection.createStatement();
+        statement.setFetchSize(getMaxResult());
```

Comment:
If `getMaxResult()` is used in statement directly, then it might not be necessary in method `getResults`